dspace-cris/api/src/main/java/org/dspace/app/cris/batch/ScriptCrisUnsubscribeEPerson.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * https://github.com/CILEA/dspace-cris/wiki/License
+ */
```

Comment:
remove the old license statement as new dspace-cris files simply follows the dspace license as mentioned in the dspace-cris root LICENSE file